### DISCUSSION TITLE
Extract contacts?

### DISCUSSION THREAD

**Discussion by NLZ** [MaxiHuHe04/iTunes-Backup-Explorer#65] [Q&A]
Anyone knows where can I see the contacts in the save?

**Comment by poorkid** [comment]
That might be HomeDomain/Library/AddressBook ? 
I was trying to restore Safari history and tabs and found the .dbs i needed in the HomeDomain folder.

**Comment by w1ggle** [comment]
Stumbled upon this and @poorkid is correct.

As of writing and ios 26, under HomeDomain/Library/AddressBook, there will be "AddressBook.sqlitedb" and "AddressBookImages.sqlitedb" files. I just renamed the file extension from .sqlitedb to .db and was able to view in VSCode with 3rd party extensions. I didn't open the image file but the first/non-image file had everything (and more) that I was looking for. In this file, I found the tables "ABPerson" and "ABPersonFullTextSearch_content" which had names and phone numbers I was looking for. 

The other tables are all supporting and tracking stuff, but I didn't dig that deep.
